contracts/echidna/mocks/EchidnaMockAddressBook.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+contract EchidnaMockAddressBook {
+    constructor() public {}
+
+    function getController() external view returns (address) {
+        return 0x00a329C0648769a73afAC7F9381e08fb43DBEA70;
```

Comment:
Will hardcoding this cause any problem?

Reply:
it is hardcoded to return the owner address for now, which it thinks is the controller. This is to test the "onlyController" functionality.

Reply:
Got it I see it now. I misunderstood it as a hardcoded accounts value that might be different if we run truffle with different seed phrase 😹